### ISSUE TITLE
can not get the Configuration

### ISSUE THREAD

**Issue by xiumin1993** [alibaba/nacos#5167]
**Describe the bug**
I declare a variable in the controller, decorated with @value
Moreover, the controller also uses the @RefreshScope annotation,
and then I configure the key in Nacos,
 but the request to get the variable value is null,

**Expected behavior**
Get the value I configured in Nacos
**Acutally behavior**
the request to get the variable value is null,.

**How to Reproduce**
 **the controller**
```@RestController
//动态刷新配置
@RefreshScope
@Configuration
public class HelloService {
    @Value("${master.name:test}")
    private String masterName;


    @GetMapping("sayHello")
    private String sayHello() {
        return "hello " + masterName;
    }

}
```
**the bootstrap.properties**
```server.port=8776
spring.application.name=service-a
spring.cloud.nacos.config.server-addr=127.0.0.1:8848
spring.cloud.nacos.discovery.server-addr=127.0.0.1:8848
```
**nacos config**
![image](https://user-images.githubusercontent.com/27759194/112028018-b7bbc180-8b72-11eb-8d12-83fb863c009e.png)
**the response**
![image](https://user-images.githubusercontent.com/27759194/112028162-d9b54400-8b72-11eb-8521-c451f6ae4d8f.png)

**Desktop :**
 - OS: win10
 - Version [ nacos-server 2.0.0, nacos-client 1.4.1]
 - Module [config]
 - SDK [spring-cloud-alibaba-dependencies-2.2.5.RELEASE,spring-boot-starter-web-2.3.2.RELEASE]

**Comment by zrlw** [comment]
@NacosPropertySource

**Comment by xiumin1993** [comment]
> https://blog.csdn.net/fly_leopard/article/details/107489937

as the blog, said  there is a wrapper bean,  but why the origin bean's default value "test" change to null

**Comment by xiumin1993** [comment]
> https://blog.csdn.net/fly_leopard/article/details/107489937

l

> Please provider the logs about nacos client to check whether nacos-client receive the new config.
> 
> If received, please move issue to SCA community.
the log when i change the value in nacos
```
2021-03-23 11:24:04.257  INFO 30532 --- [-127.0.0.1_8848] c.a.n.client.config.impl.ClientWorker    : [fixed-127.0.0.1_8848] [polling-resp] config changed. dataId=service-a, group=DEFAULT_GROUP
2021-03-23 11:24:04.258  INFO 30532 --- [-127.0.0.1_8848] c.a.n.client.config.impl.ClientWorker    : get changedGroupKeys:[service-a+DEFAULT_GROUP]
2021-03-23 11:24:04.274  INFO 30532 --- [-127.0.0.1_8848] c.a.nacos.client.config.utils.JvmUtil    : isMultiInstance:false
2021-03-23 11:24:04.275  INFO 30532 --- [-127.0.0.1_8848] c.a.n.client.config.impl.ClientWorker    : [fixed-127.0.0.1_8848] [data-received] dataId=service-a, group=DEFAULT_GROUP, tenant=null, md5=96527b918278014153f9f35e308a7704, content=master.name=tom, type=properties
2021-03-23 11:24:04.275  INFO 30532 --- [-127.0.0.1_8848] c.a.nacos.client.config.impl.CacheData   : [fixed-127.0.0.1_8848] [notify-context] dataId=service-a, group=DEFAULT_GROUP, md5=96527b918278014153f9f35e308a7704
2021-03-23 11:24:05.990  WARN 30532 --- [-127.0.0.1_8848] c.a.c.n.c.NacosPropertySourceBuilder     : Ignore the empty nacos configuration and get it based on dataId[service-a.properties] & group[DEFAULT_GROUP]
2021-03-23 11:24:05.990  INFO 30532 --- [-127.0.0.1_8848] b.c.PropertySourceBootstrapConfiguration : Located property source: [BootstrapPropertySource {name='bootstrapProperties-service-a.properties,DEFAULT_GROUP'}, BootstrapPropertySource {name='bootstrapProperties-service-a,DEFAULT_GROUP'}]
2021-03-23 11:24:05.992  INFO 30532 --- [-127.0.0.1_8848] o.s.boot.SpringApplication               : No active profile set, falling back to default profiles: default
2021-03-23 11:24:06.001  INFO 30532 --- [-127.0.0.1_8848] o.s.boot.SpringApplication               : Started application in 1.722 seconds (JVM running for 1205.22)
2021-03-23 11:24:06.028  INFO 30532 --- [-127.0.0.1_8848] o.s.c.e.event.RefreshEventListener       : Refresh keys changed: [master.name]
2021-03-23 11:24:06.028  INFO 30532 --- [-127.0.0.1_8848] c.a.nacos.client.config.impl.CacheData   : [fixed-127.0.0.1_8848] [notify-ok] dataId=service-a, group=DEFAULT_GROUP, md5=96527b918278014153f9f35e308a7704, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@53dc96c0 
2021-03-23 11:24:06.028  INFO 30532 --- [-127.0.0.1_8848] c.a.nacos.client.config.impl.CacheData   : [fixed-127.0.0.1_8848] [notify-listener] time cost=1753ms in ClientWorker, dataId=service-a, group=DEFAULT_GROUP, md5=96527b918278014153f9f35e308a7704, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@53dc96c0 

```

**Comment by KomachiSion** [comment]
2021-03-23 11:24:04.275  INFO 30532 --- [-127.0.0.1_8848] c.a.n.client.config.impl.ClientWorker    : [fixed-127.0.0.1_8848] [data-received] dataId=service-a, group=DEFAULT_GROUP, tenant=null, md5=96527b918278014153f9f35e308a7704, content=master.name=tom, type=properties
2021-03-23 11:24:04.275  INFO 30532 --- [-127.0.0.1_8848] c.a.nacos.client.config.impl.CacheData   : [fixed-127.0.0.1_8848] [notify-context] dataId=service-a, group=DEFAULT_GROUP, md5=96527b918278014153f9f35e308a7704

nacos-client has received the new configuration. so nacos has finished the workflow. You may move this issue to SCA to find out the answer.
